layers/+lang/javascript/README.org

```diff
@@ -149,3 +159,14 @@ You can check more [[https://github.com/mooz/js-doc/][here]]
 | Key Binding | Description                        |
 |-------------+------------------------------------|
 | ~SPC m h p~ | Get the path of the value at point |
+
+** REPL (skewer-mode)
+
+| Key Binding | Description                                                             |
+|-------------+-------------------------------------------------------------------------|
+| ~SPC m s i~ | prepares the skewer environment - run this before running REPL commands |
+| ~SPC m s r~ | starts/goes to the skewer REPL                                          |
```

Comment:
I set ```SPC m s i``` to the start/switch to REPL command, to respect conventions. The ```SPC m s s``` is now used for preparing the REPL environment.